### DISCUSSION TITLE
如何实现lazyload的disqus

### DISCUSSION THREAD

**Discussion by AprilNEA** [ppoffice/hexo-theme-icarus#904] [Q&A] [answered]
https://blog.skk.moe/post/prevent-disqus-from-slowing-your-site/
应该在主题内哪处应用上述博客内的代码？

**Comment by ppoffice** [comment, marked as answer]
@GreenSulley 
https://github.com/ppoffice/hexo-component-inferno/blob/0.13.0/src/view/comment/disqus.jsx
https://ppoffice.github.io/hexo-theme-icarus/uncategorized/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98/#%E5%B8%83%E5%B1%80
